data/src/test/java/org/apache/iceberg/data/TestDataFileIndexStatsFilters.java

```diff
@@ -163,10 +163,11 @@ public void testPositionDeletePlanningPathFilter() throws IOException {
       tasks = Lists.newArrayList(tasksIterable);
     }
 
-    Assert.assertEquals("Should produce one task", 1, tasks.size());
+    Assertions.assertThat(tasks.size()).isEqualTo(1).as("Should produce one task");
     FileScanTask task = tasks.get(0);
-    Assert.assertEquals(
-        "Should not have delete file, filtered by file_path stats", 0, task.deletes().size());
+    Assertions.assertThat(task.deletes().size())
+        .isEqualTo(0)
+        .as("Should not have delete file, filtered by file_path stats");
```

Comment:
`.as()` needs to come before the final assertion check. Also this should be `assertThat(task.deletes).as(..).hasSize(..)`